source/renderer/app/components/notifications/Notification.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+import React, { Component, Fragment } from 'react';
+import type { Node } from 'react';
+import SVGInline from 'react-svg-inline';
+import classNames from 'classnames';
+import styles from './Notification.scss';
+import closeCross from '../../assets/images/close-cross.inline.svg';
+
+export type NotificationMessageProps = {
+  icon?: string,
+  clickToClose?: boolean,
+  hasCloseButton?: boolean,
+  hasEllipsis?: boolean,
+  themeOverride?: 'grey', // if left empty, the noticiation will have its normal colors
+  labelValues?: Object,
+  hasSpinner?: boolean,
+};
+
+type Props = {
+  ...$Exact<NotificationMessageProps>,
+  children?: Node,
+  onClose?: Function,
+  isVisible: boolean,
+  order?: 'auto' | number | 'initial' | 'inherit',
+};
+
+export default class Notification extends Component<Props> {
+  static defaultProps = {
+    order: 9999999999999999,
```

Comment:
Maybe extract to external const variable and reuse it from there?

Reply:
@DeeJayElly  ✅ Replaced `order` by `index` received by the container, so the last notification is always on top.